stylelint.config.js

```diff
@@ -32,6 +32,7 @@ module.exports = {
     'selector-pseudo-class-no-unknown': true,
     'selector-pseudo-element-no-unknown': true,
     'comment-no-empty': true,
+    'max-line-length': 200,
```

Comment:
200 is a big number for the maximum line length. Is there a particular reason 200 was chosen? Common values for coding are 80 (maybe too little) or 120. Maybe we can go with 120 if 80 isn't good enough? I couldn't find out what was the default one used by Stylelint if no none was specified.

Reply:
I need this in order to add this mixin: [input-focus-ring](https://github.com/vtex-sites/base.store/pull/224/files#diff-866f5ecb98e17251bb80a1e6b2acf44f290b595a7ff29dbf29f15d6be250192aR25
)

Reply:
Ah, I see now.

I see 2 options then:
1. Use line breaks for that line, e.g.
```css
box-shadow: 0 0 0 1px var(--bg-neutral-lightest),
    0 0 0 var(--border-width-2) var(--bg-focus-ring),
    inset 0 0 0 var(--border-width-0) var(--color-border-input-selected);
```

2. Temporarily disable Stylelint for that specific line, e.g.
```scss
@mixin focus-ring {
  outline: none;
  /* stylelint-disable-next-line max-line-length */
  box-shadow: 0 0 0 1px var(--bg-neutral-lightest), 0 0 0 var(--border-width-2) var(--bg-focus-ring);
}
```

I like the #1 better. What do you think? Then we don't need to modify this config.

Reply:
I like the #1 better as well 😄

Reply:
The first one shows this error:
![image](https://user-images.githubusercontent.com/11613011/150128874-6cc2024e-8dc3-4017-a132-e554f622cd96.png)
And when I try to add the extra line requested and save it (see image below), it moves the line back to the beginning.
<img width="562" alt="Screen Shot 2022-01-19 at 09 25 50" src="https://user-images.githubusercontent.com/11613011/150129084-29521362-cef6-4fcd-9543-17ed2fc1738b.png">

Reply:
Hmm, there seem to be some conflicting Stylelint rules. I was able to commit after the following change to [declaration-colon-space-after](https://stylelint.io/user-guide/rules/list/declaration-colon-space-after/#always-single-line):
![CleanShot 2022-01-19 at 15 03 43](https://user-images.githubusercontent.com/381395/150188250-f935152f-5210-46cb-ab75-6daa31d84e3c.png)